### PR TITLE
[Table Vis]Switch to the new table plugin

### PR DESCRIPTION
**Pull request by conglei**
### CATEGORY

Choose one

- [x] Enhancement (new features, refinement)

### SUMMARY

This PR is to use the new Table plugin to improve the rendering performance. The new plugin is built based on `react-virtualized`.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
table-demo

### REVIEWERS

@kristw

### CODE REVIEW

File: superset/assets/src/visualizations/presets/MainPreset.js [hunk 1]
```diff
@@ -36,7 +36,8 @@ import PivotTableChartPlugin from '@superset-ui/legacy-plugin-chart-pivot-table'
 import RoseChartPlugin from '@superset-ui/legacy-plugin-chart-rose';
 import SankeyChartPlugin from '@superset-ui/legacy-plugin-chart-sankey';
 import SunburstChartPlugin from '@superset-ui/legacy-plugin-chart-sunburst';
-import TableChartPlugin from '@superset-ui/legacy-plugin-chart-table';
+// import TableChartPlugin from '@superset-ui/legacy-plugin-chart-table';
```
Comment (kristw) [inline superset/assets/src/visualizations/presets/MainPreset.js]: remove

**Review by kristw — APPROVED** [review]

**Comment by kristw** [comment]
So excited!!!

**Review by etr2460 — COMMENTED** [review]
can you include screenshots and a test plan? I remember when we merged this before we had to revert because not all edge cases were checked

**Comment by conglei** [comment]
@etr2460. Will get some screenshot for the test cases I did.

**Comment by kristw** [comment]
@conglei There seems to be an issue with your lock file.

**Comment by conglei** [comment]
Looking

**Review by etr2460 — APPROVED** [review]
let's do it!

**Comment by kristw** [comment]
woo hoo

**Comment by conglei** [comment]
I don't have the permission to merge ..
